auth/index.py

```diff
@@ -105,6 +105,74 @@ def verify_token_uid(id_token):
     print(uid)
     firebase_admin.delete_app(default_app)
 
+def get_user(uid):
+    # [START get_user]
+    from firebase_admin import auth
+
+    user = auth.get_user(uid)
+    print 'Successfully fetched user data: {0}'.format(user.uid)
+    # [END get_user]
+
+def get_user_by_email():
+    email = 'user@example.com'
+    # [START get_user_by_email]
+    from firebase_admin import auth
```

Comment:
I am fine either way, but I don't think we need to show the `import` in every snippet.  Maybe just in one top-of-page snippet.